release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2674,3 +2674,30 @@ The minimum storage required to install an {product-title} cluster has decreased
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-8-13"]
+=== RHBA-2021:3632 - {product-title} 4.8.13 bug fix and security update
+
+Issued: 2021-09-27
+
+{product-title} release 4.8.13, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2021:3632[RHBA-2021:3632] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2021:3631[RHSA-2021:3631] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6363171[{product-title} 4.8.13 container image list]
+
+[id="ocp-4-8-13-features"]
+==== Features
+
+* Kubernetes v1.21.4 is now available. More information can be found in the following changelogs: link:https://github.com/kubernetes/kubernetes/blob/release-1.21/CHANGELOG/CHANGELOG-1.21.md#changelog-since-v1213[v1.21.4], link:https://github.com/kubernetes/kubernetes/blob/release-1.21/CHANGELOG/CHANGELOG-1.21.md#changelog-since-v1212[v1.21.3], and link:https://github.com/kubernetes/kubernetes/blob/release-1.21/CHANGELOG/CHANGELOG-1.21.md#changelog-since-v1211[v1.21.2]. 
```

Comment:
Not critical, but we usually don't put a "v" in front of the versions. I'd just say "Kubernetes 1.21.4..."